pkg/git/fetch_test.go

```diff
@@ -79,7 +79,7 @@ func testGitCheckout(t *testing.T, when spec.G, it spec.S) {
 
 		it("returns error on non-existent ref", func() {
 			err := fetcher.Fetch(testDir, "https://github.com/git-fixtures/basic", "doesnotexist", metadataDir)
-			require.EqualError(t, err, "resolving revision: reference not found")
+			require.EqualError(t, err, "unable to fetch references for repository: couldn't find remote ref \"doesnotexist\"")
```

Comment:
Error message has changed. Updated to newly expected error message such that the tests pass.